inc/container.class.php

```diff
@@ -1139,7 +1139,7 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
         if (isset($itemtypes[$item->getType()])) {
             $tabs_entries = [];
             $container    = new self();
-            foreach ($itemtypes[$item->getType()] as $tab_name => $tab_label) {
+            foreach ($itemtypes[$item->getType()] as $tab_name) {
```

Comment:
It seems that `$itemtypes[$item->getType()]` can return either a string or an array. The previous code handled the array case, while this one only handles the string case. It needs to work with both.

Reply:
tab label is no longer used as I now look for the value in the base directly